taxonomies/additives_classes.txt

```diff
@@ -221,7 +221,7 @@ ga: Frithocsaídeoir
 hr: antioksidans, antioksidansi, antioksidant, antioksidat, antioksindans
 hu: Antioxidáns, antioxidánsok
 it: Antiossidante, antiossidanti, agente antiossidante, agenti antiossidanti
-ja: 酸化防止剤
+ja: 酸化防止剤,酸化防 止剤
```

Comment:
Placing a space in a word does not make sense. They should be corrected instead.